AnkiDroid/src/main/java/com/ichi2/libanki/sched/AbstractSched.java

```diff
@@ -232,4 +232,10 @@ public String toString() {
     public interface CountMethod {
         int operation(long did, int lim);
     }
+
+    // Should be called in background, preload next question/answer.
+    // Synchronzide, so that if the nextCard is already computed, it
+    // does not get overridden.
+    // Not in libanki. In AnkiDroid to save time
```

Comment:
```suggestion
    // Preload next question/answer.
    // Not in libanki. In AnkiDroid to improve interactive responsiveness
```

Just comment tweaks for precision -

- the synchronization is an implementation detail and it is not actually synchronized here in the definition
- it actually spends the same amount of application time, possibly more if pre-computation is thrown away so it is not really saving time, but it's not useless at all - it improves responsiveness to the user a lot

Reply:
I wrote the comment thinking of myself as a user. As a user who hates waiting even a second for the next question. And it does save me time.
I never thought for a second that it was saving computation time; however that's not something I cared about here

I will however admit that my writting may be ambiguous, especially since I've been known to reduce computation time of a lot of task in those past two weeks.